conda_env/cli/main_create.py

```diff
@@ -81,6 +88,8 @@ def execute(args, parser):
         common.error_and_exit(str(e), json=args.json)
 
     prefix = common.get_prefix(args, search=False)
+    if args.force and not is_root_prefix and os.path.exists(prefix):
```

Comment:
`is_root_prefix` is a function that expects a prefix as an argument. Think you are missing that here.
Take a look at https://github.com/conda/conda/blob/master/conda/plan.py#L312
See usage in https://github.com/conda/conda-env/blob/develop/conda_env/cli/main_remove.py#L46